frontend/src/components/design-system/containers/ContainerComponent.vue

```diff
@@ -1,17 +1,38 @@
 <script setup lang="ts">
+import { computed } from 'vue';
+
 export interface Props {
-  padding?: 'small' | 'large';
+  padding?: boolean;
```

Comment:
Není tu tato proměnná navíc? Si vystačíš jen s `paddingSize`, pokud default dáš undefined.

Reply:
Ta o řádek výš. Mažu.

Reply:
Nebo to tak můžeme dělat, jenom to pak vyžaduje při každém použití té komponenty ji nastavit tu hodnotu. @kyjanond?

Reply:
kdyz uz to mas jako optional tak nepotrebujes boolean prece. dej padding?: 'small'|'large' a mas to. kdyz je undefined tak tam neni. job done